invopop/gobl.builder
# Use custom `JSONWorker` class

Use custom `JSONWorker` so we can import our own up-to-date version of `vscode-json-languageservice` which has support for JSON Schema Draft 2020-12.

Fixes #2 #patch

### QA

To test, visit https://deploy-preview-21--gobl-build.netlify.app/ and type/paste a JSON doc that references a Draft 2020-12 schema. E.g.:

```json
{
    "$schema": "https://json-schema.org/learn/examples/address.schema.json"
}
```

You **shouldn't** see the warning described in #21.

(Note: You'll see an error stating that the `$schema` value is not right — that's an expected error, ignore for this QA)
